isis/tests/FunctionalTestsCaminfo.cpp

```diff
@@ -0,0 +1,313 @@
+#include "Cube.h"
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+
+#include "caminfo.h"
+
+#include "gtest/gtest.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/caminfo.xml").expanded();
+
+
+TEST_F(DefaultCube, FunctionalTestCaminfoCsv) {
+    QString outFileName = tempDir.path() + "/outTemp.csv";
+    QVector<QString> args = {"from="+ testCube->fileName(),  "to="+outFileName,
+        "FORMAT=flat", "APPEND=false", "STATISTICS=true", "CAMSTATS=true",
+        "GEOMETRY=true", "spice=true"};
+
+    UserInterface options(APP_XML, args);
+    try {
+       caminfo(options);
+    }
+    catch (IException &e) {
+        FAIL() << "Unable to open image: " << e.what() << std::endl;
+    }
+
+    CSVReader::CSVAxis csvLine;
+    // Access the "Header" information first
+    CSVReader header = CSVReader(outFileName,
+                                 false, 0, ',', false, true);
+
+    // Validate the header information is correct
+    csvLine = header.getRow(1);
+    ASSERT_EQ(csvLine[0], "caminfo");
```

Comment:
Can you change the asserts to expects in this test? When using assert, if one fails, the whole test fails. With expect, if one fails, the test continues. I generally use assert for something that is crucial for the test, like the size of an array so we don't go out of bounds.